### ISSUE TITLE
Apply to object not in training dataset

### ISSUE THREAD

**Issue by davidemariafabris** [j96w/DenseFusion#58]
Hi,

I am currently working at my Master's Thesis on 3D Object Pose Estimation and I found your application to be interesting. For the specific case I am working on, I was wondering whether it would be possible to directly apply the entire framework Segmentation (trained by me on YCB) + Estimation (trained by you on YCB) + Refining (trained by you on YCB) , without any further training, on an **object not in the YCB training dataset**.

I am able to provide the architecture with all the required input parameters (RGB, depth, .ply, camera intrinsics, depth units), still I was asking my self if the **unseen object** pose would be estimated properly.

If this is not the case, may I ask if:

1 - both Segmentation and Estimation+Refining;
2 - only Segmentation;
3 - only Estimation+Refining;

need to be trained on a new dataset containing the necessary information related to the chosen **object not in the training dataset**?

Thank you for your precious help,

Davide

**Comment by BarbeBleue** [comment]
I used SegNet and DenseFusion with an other object and to this end I had to train both networks with a dataset I did to this end containing rgb, depth and labels images as well as ground truth values.

**Comment by j96w** [comment]
Hi, a follow up work will appear in the next few months that will mainly discuss how to apply to unseen objects. Please wait and see.
